CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## DLA-Future 0.7.3
+
+### Bug fixes
+
+- Changed C ScaLAPACK API indexing convention to 1-based for partial eigenspectrum. (#1248)
+
+## DLA-Future 0.7.2
+
+Not released.
+
```

Comment:
Not sure if we want to add this or not, but might be useful as information for users questioning why there is no `0.7.2`?

Reply:
I like this, let's keep it.